### ISSUE TITLE
Bug: Uncaught TypeError: Class constructor pe cannot be invoked without 'new'

### ISSUE THREAD

**Issue by edev2020** [pixijs/pixijs#9107]
### Current Behavior

DragonBones pixijs plugin is no longer working with v7 however it works fine with [v6.5.8].
Throwing error: **Uncaught TypeError: Class constructor pe cannot be invoked without 'new'**
"pe" is a class inside pixi.min.js: `class An extends pe`

### Expected Behavior

No error should show normally.

### Steps to Reproduce

Get dragonbones pixijs plugin from here: https://github.com/DragonBones/DragonBonesJS/tree/master/Pixi/5.x/out
Actual plugin link: https://github.com/DragonBones/DragonBonesJS/blob/master/Pixi/5.x/out/dragonBones.min.js
Write this code in your pixi application: `const factory = dragonBones.PixiFactory.factory;`

### Environment

- **`pixi.js` version**: *7.1.1*
- **Browser & Version**: *Chrome 109.0.5414.75*
- **OS & Version**: *Windows 10*

### Additional Information

![Screenshot_2](https://user-images.githubusercontent.com/70892011/214419165-e9b83dc8-a23c-4338-b147-f26ee42e664b.png)
![Screenshot_1](https://user-images.githubusercontent.com/70892011/214419170-7a6db231-2c64-4872-9caf-c4b8e1bee8fb.png)

**Comment by SuperSodaSea** [comment]
For pixi.js@7 we [output ES2017 code for browser bundles by default](https://github.com/pixijs/pixijs/wiki/v7-Migration-Guide#-output-es2020-modules-and-es2017-browser), so the classes are not transpiled. The dragonBones.js, however, do transpile the class, and these two are not compatible.

Document of [@babel/plugin-transform-classes](https://babeljs.io/docs/en/babel-plugin-transform-classes):

> Babel transpiles classes using SuperClass.apply(/* ... */), but native classes aren't callable and thus throw in this case.

You can try to use Babel to transpile pixi.js with `@babel/plugin-transform-classes` enabled to transpile the class. However, I can't guarantee that it will work properly after doing so, as dragonBones.js may have other incompatibilities with pixi.js@7. Anyway, you can try it first.

**Comment by edev2020** [comment]
> For pixi.js@7 we [output ES2017 code for browser bundles by default](https://github.com/pixijs/pixijs/wiki/v7-Migration-Guide#-output-es2020-modules-and-es2017-browser), so the classes are not transpiled. The dragonBones.js, however, do transpile the class, and these two are not compatible.
> 
> Document of [@babel/plugin-transform-classes](https://babeljs.io/docs/en/babel-plugin-transform-classes):
> 
> > Babel transpiles classes using SuperClass.apply(/* ... */), but native classes aren't callable and thus throw in this case.
> 
> You can try to use Babel to transpile pixi.js with `@babel/plugin-transform-classes` enabled to transpile the class. However, I can't guarantee that it will work properly after doing so, as dragonBones.js may have other incompatibilities with pixi.js@7. Anyway, you can try it first.

Thank you very much works perfectly.

**Comment by kulabros** [comment]
Hello guys, thanks a lot for this information. I'm facing the same issue, I've started updating my big project from v6 to v7.

I'm trying to figure out how to add the babel plugin in our build, I tried it and it didn't worked, I guess I'm doing something wrong, but anyway, let me ask. I also tried adding target: es6 into typescript config.. Both did not helped in my case.

Is there some other way how to solve this problem with pixi-spine constructor? Can I write somehow custom spine constructor that wil work, or can I change some function/class definition, to achieve it's working? Or is the babel transpiling the only way?

I'm using pixi-spine from npm deps. imported at the top level and also I have custom bundle base + 4.1 runtime.
